vike/client/client-routing-runtime/prefetch/getPrefetchSettings.ts

```diff
@@ -1,29 +1,39 @@
 export { getPrefetchSettings }
+export { PrefetchPageContext }
 
-import type { PrefetchStaticAssets } from '../../../shared/types/PrefetchStaticAssets.js'
+import type { PrefetchWhen, PrefetchExpire } from '../../../shared/types/Prefetch.js'
 import { assert, assertUsage, assertInfo, assertWarning, isPlainObject } from '../utils.js'
 
 type PageContextPrefetch = {
   exports: Record<string, unknown>
 }
 
+type PrefetchPageContext = {
+  when?: Exclude<PrefetchWhen, 'viewport'>
+  expire?: number
+}
+
 type PrefetchSettings = {
-  prefetchStaticAssets: PrefetchStaticAssets
+  prefetchStaticAssets: PrefetchWhen
+  prefetchPageContext: PrefetchPageContext
 }
 
 function getPrefetchSettings(pageContext: PageContextPrefetch, linkTag: HTMLElement): PrefetchSettings {
   let prefetchStaticAssets = getPrefetchStaticAssets(pageContext, linkTag)
-  if (prefetchStaticAssets === 'viewport' && import.meta.env.DEV) {
+  let prefetchPageContext = getPrefetchPageContext(pageContext, linkTag)
+  if (prefetchStaticAssets === 'viewport' && process.env.NODE_ENV === 'development') {
```

Comment:
Why did you change it to `process.env.NODE_ENV`? I'm not sure Vite supports `process.env.NODE_ENV`.

Reply:
Even though I didn't add `import.meta.env.DEV`, If I had kept that part, the CI test would have failed.
https://github.com/vikejs/vike/actions/runs/9136157470/job/25124574955

Reply:
I see. I suggest we have a look at it again after we're happy with the rest. Marking as resolved in the meantime.